Formula/gnuradio.rb

```diff
@@ -219,6 +211,7 @@ def main(top_block_cls=top_block, options=None):
 
       main()
     EOS
-    system Formula["python@3.9"].opt_bin/"python3", testpath/"test.py"
+    # system Formula["python@3.9"].opt_bin/"python3", testpath/"test.py"
+    system libexec/"venv/bin/python3", testpath/"test.py"
```

Comment:
Not sure this is a good idea. If the current test needs to use the venv Python, then we should instead fix the `install` method so that it doesn't, as this seems to break things for existing users otherwise.

Reply:
Related change: https://github.com/gnuradio/gnuradio/commit/de9b45677c98c089c4a81c9e201446c996f8e972

Originally, `GR_PYTHON_DIR` would be set to `#{lib}/python3.9/site-packages`.

Now, it prioritizes values in `site.getsitepackages()`. When run using virtualenv python, this means `#{libexec}/venv/python3.9/site-packages` is picked.

---

One option would be to try to restore original behavior by modifying `PYTHON_EXECUTABLE` or `GR_PYTHON_DIR`.

Another option would be to install into venv and then change the `.pth` files to match.